internal/usage/manager.go

```diff
@@ -64,3 +68,15 @@ func Unmarshal(str string) (FeatureUsage, error) {
 	err := json.Unmarshal([]byte(str), &result)
 	return result, err
 }
+
+func UsageKeyToName(featureKey string) (name string) {
+	name = featureKey
+	return
+}
+
+func UsageKeyToID(featureKey string) (id string) {
+	name := UsageKeyToName(featureKey)
+	r := strings.NewReplacer(" ", "_", ".", "")
+	id = strings.ToUpper(r.Replace(name))
+	return
```

Comment:
nit:
I prefer to``return id`` explicitly 

but not critical

Reply:
👍🏽